addons/mail/models/mail_notification.py

```diff
@@ -45,6 +45,7 @@ class MailNotification(models.Model):
         ("unknown", "Unknown error"),
         # mail
         ("mail_bounce", "Bounce"),
+        ("mail_detected_as_spam", "Detected As Spam"),
```

Comment:
Let us save space in DB :D 
```suggestion
        ("mail_spam", "Detected As Spam"),
```